src/BinSkim.Sdk/EnvironmentResources.resx

```diff
@@ -0,0 +1,123 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I do not know how to review this, you might need a second set of eyes for it.

Reply:
the .net resource file is auto gen,
the only thing that is worth looking is the `DefaultTelemetryConnectionString` at the end.